package.json

```diff
@@ -13,21 +13,17 @@
     "test": "npm run lint",
     "lint": "eslint ."
   },
-  "devDependencies": {
-    "babel-eslint": "^7.2.2",
-    "eslint": "^3.19.0",
-    "eslint-plugin-import": "^2.2.0",
-    "eslint-plugin-jsx-a11y": "^5.0.3",
-    "eslint-plugin-react": "^7.0.1"
-  },
-  "peerDependencies": {
-    "babel-eslint": "^7.2.2",
-    "eslint": "^3.19.0",
-    "eslint-plugin-import": "^2.2.0",
-    "eslint-plugin-jsx-a11y": "^5.0.3",
-    "eslint-plugin-react": "^7.0.1"
-  },
+  "devDependencies": {},
+  "peerDependencies": {},
   "dependencies": {
-    "eslint-config-airbnb": "15.0.1"
+    "babel-eslint": "8.1.2",
+    "eslint": "4.14.0",
```

Comment:
* Add as dependencies so each project doesn't have to remember to install it
* Lock versions (instead of using `^`) to ensure all of our projects are using the same rules. If/when we want to bump versions of lint-related packages we'll do so here.